### PR TITLE
Dev cart page

### PR DESCRIPTION
**Pull request by kmangan89**
With these changes, the cart page will now reflect the designs proposed by Irish Titan (image below). The header, footer, and navigation along with links to other parts of the site (like on the individual products or on the header image) have been removed. Additionally, custom scss files have been added to override some of the cornerstone theme. 

#### Screenshots (if appropriate)
Screen Shot 2019-12-05 at 2 48 53 PM

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -233,16 +233,16 @@
     "optimizedCheckout-logo-size": "250x100",
     "optimizedCheckout-logo-position": "left",
     "optimizedCheckout-headingPrimary-color": "#333333",
-    "optimizedCheckout-headingPrimary-font": "Google_Montserrat_700",
+    "optimizedCheckout-headingPrimary-font": "Google_Questrial_700",
```
Comment (TastefullySimple) [inline config.json]: I assume all these over 400 will default to 400?

File: package-lock.json [hunk 2]
```diff
@@ -4713,6 +4722,15 @@
       "integrity": "sha1-rT/0yG7C0CkyL1oCw6mmBslbP1k=",
       "dev": true
     },
+    "encoding": {
+      "version": "0.1.12",
+      "resolved": "https://registry.npmjs.org/encoding/-/encoding-0.1.12.tgz",
+      "integrity": "sha1-U4tm8+5izRq1HsMjgp0flIDHS+s=",
```
Comment (TastefullySimple) [inline package-lock.json]: Is there a sha512 option?

File: package-lock.json [hunk 3]
```diff
@@ -4826,6 +4844,12 @@
       "integrity": "sha1-vfpzUplmTfr9NFKe1PhSKidf6lY=",
       "dev": true
     },
+    "err-code": {
+      "version": "1.1.2",
+      "resolved": "https://registry.npmjs.org/err-code/-/err-code-1.1.2.tgz",
+      "integrity": "sha1-BuARbTAo9q70gGhJ6w6mp0iuaWA=",
```
Comment (TastefullySimple) [inline package-lock.json]: Is there a sha512 option?

File: package-lock.json [hunk 4]
```diff
@@ -4947,6 +4971,15 @@
       "integrity": "sha512-HJDGx5daxeIvxdBxvG2cb9g4tEvwIk3i8+nhX0yGrYmZUzbkdg8QbDevheDB8gd0//uPj4c1EQua8Q+MViT0/w==",
       "dev": true
     },
+    "es6-promisify": {
+      "version": "5.0.0",
+      "resolved": "https://registry.npmjs.org/es6-promisify/-/es6-promisify-5.0.0.tgz",
+      "integrity": "sha1-UQnWLz5W6pZ8S2NQWu8IKRyKUgM=",
```
Comment (TastefullySimple) [inline package-lock.json]: Is there a sha512 option?

File: package-lock.json [hunk 5]
```diff
@@ -9342,6 +9355,42 @@
...
+          "integrity": "sha512-OX8XqP7/1a9cqkxYw2yXss15f26NKWBpDXQd0/uK/KPqdQhxbPa994hnzjcE2VqQpDslf55723cKPUOGSmMY3g==",
+          "dev": true,
+          "requires": {
+            "ms": "2.0.0"
+          }
+        },
+        "ms": {
+          "version": "2.0.0",
+          "resolved": "https://registry.npmjs.org/ms/-/ms-2.0.0.tgz",
+          "integrity": "sha1-VgiurfwAvmwpAd9fmGF4jeDVl8g=",
```
Comment (TastefullySimple) [inline package-lock.json]: Is there a sha512 option?

File: package-lock.json [hunk 6]
```diff
@@ -9387,6 +9436,15 @@
         }
       }
     },
+    "humanize-ms": {
+      "version": "1.2.1",
+      "resolved": "https://registry.npmjs.org/humanize-ms/-/humanize-ms-1.2.1.tgz",
+      "integrity": "sha1-xG4xWaKT9riW2ikxbYtv6Lt5u+0=",
```
Comment (TastefullySimple) [inline package-lock.json]: Is there a sha512 option?

File: package-lock.json [hunk 7]
```diff
@@ -9673,6 +9731,12 @@
       "integrity": "sha1-EEqOSqym09jNFXqO+L+rLXo//bY=",
       "dev": true
     },
+    "ip": {
+      "version": "1.1.5",
+      "resolved": "https://registry.npmjs.org/ip/-/ip-1.1.5.tgz",
+      "integrity": "sha1-vd7XARQpCCjAoDnnLvJfWq7ENUo=",
```
Comment (TastefullySimple) [inline package-lock.json]: Is there a sha512 option?

File: package-lock.json [hunk 8]
```diff
@@ -14655,6 +14923,12 @@
           "integrity": "sha1-xQYbbg74qBd15Q9dZhUb9r83EQc=",
           "dev": true
         },
+        "ansi-styles": {
+          "version": "2.2.1",
+          "resolved": "https://registry.npmjs.org/ansi-styles/-/ansi-styles-2.2.1.tgz",
+          "integrity": "sha1-tDLdM1i2NM914eRmQ2gkBTPB3b4=",
```
Comment (TastefullySimple) [inline package-lock.json]: Is there a sha512 option?

File: package-lock.json [hunk 9]
```diff
@@ -14673,14 +14947,52 @@
           "integrity": "sha1-+cjBN1fMHde8N5rHeyxipcKGjEA=",
           "dev": true
         },
+        "assert-plus": {
+          "version": "0.2.0",
+          "resolved": "https://registry.npmjs.org/assert-plus/-/assert-plus-0.2.0.tgz",
+          "integrity": "sha1-104bh+ev/A24qttwIfP+SBAasjQ=",
```
Comment (TastefullySimple) [inline package-lock.json]: Is there a sha512 option?

File: package-lock.json [hunk 10]
```diff
@@ -14673,14 +14947,52 @@
...
           "integrity": "sha1-TYqBYg1ZWHkbW5j4AtMgd3bpVQk=",
           "dev": true,
           "requires": {
             "dezalgo": "^1.0.2"
           }
         },
+        "aws-sign2": {
+          "version": "0.6.0",
+          "resolved": "https://registry.npmjs.org/aws-sign2/-/aws-sign2-0.6.0.tgz",
+          "integrity": "sha1-FDQt0428yU0OW4fXY81jYSwOeU8=",
```
Comment (TastefullySimple) [inline package-lock.json]: Is there a sha512 option?

File: package-lock.json [hunk 11]
```diff
@@ -14673,14 +14947,52 @@
...
+          "dev": true
+        },
+        "bl": {
+          "version": "1.1.2",
+          "resolved": "https://registry.npmjs.org/bl/-/bl-1.1.2.tgz",
+          "integrity": "sha1-/cqHGplxOqANGeO7ukHER4emU5g=",
```
Comment (TastefullySimple) [inline package-lock.json]: Is there a sha512 option?

File: config.json (hunk 1 above)
Comment (kmangan89) [reply config.json]: Thanks for that catch! The font weights have now been updated to 400.

**Pull request merged**